### DISCUSSION TITLE
Use of camera

### DISCUSSION THREAD

**Discussion by limi0129** [bitcraze/discussions#1258] [Q&A]
We want to use Crazyflie along with the vertical ranging expansion deck and monocular camera to complete the autonomous navigation model test of our paper. The camera selected your product ai camera expansion deck. During the installation process, it was found that the Crazyflie self-test failed to install two expansion decks at the same time. We also found in the product introduction of the ai camera expansion deck that it cannot be used with other expansion decks. Then, we would like to ask whether there is a solution, such as replacing high-power battery. In addition, we want to know what way to enable the ai camera expansion deck to mount both vertical ranging and the monocular camera?

**Comment by knmcguire** [comment]
Hi!

That is strange. I'm assuming that you are talking about the[ z-ranging deck](https://www.bitcraze.io/products/z-ranger-deck-v2/) and the [aideck](https://www.bitcraze.io/products/ai-deck/)? They should be compatible according to the deck--deck graph: https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck---deck

You are probably referring this line in the[ aideck getting started tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/):

> Make sure that only the AI-deck is attached to the Crazyflie, with no other deck.

This is only relevant for if you'd like to flash the aideck with the latest firmware over air, then no other deck should be connected. During operation the 2 decks that you mentioned should be compatible. 

After such a failure (red blinking), could you connect to the Crazyflie through the cfclient and copy the console tab output here?
